cou/steps/plan.py

```diff
@@ -84,6 +86,33 @@ def verify_supported_series(analysis_result: Analysis) -> None:
         )
 
 
+def verify_os_versions_control_plane(apps: list[OpenStackApplication]) -> None:
```

Comment:
This does not make any sense here, it should be part of the application and called by `upgrade_plan_sanity_checks` introduced in #261.

Like this we need to integrate over apps (we are doing that when we call generate upgrade plan), check that for some app it should be skipped (for subordinate we are skipping it).